source/request-response/subscription_manager.c

```diff
@@ -326,6 +343,12 @@ static const char *s_rr_subscription_event_type_to_c_str(enum aws_rr_subscriptio
 
         case ARRSET_STREAMING_SUBSCRIPTION_HALTED:
             return "StreamingSubscriptionHalted";
+
+        case ARRSET_UNSUBSCRIBE_COMPLETE:
+            return "UnsubscribeComplete";
```

Comment:
Trivial: the `s_aws_mqtt_request_response_operation_state_to_c_str` returns in the form of "PENDING_RESPONSE" and here we're returning in the form of pascal case... nvm, while writing this I found that we're using snake case elsewhere and there's no conformity =P